src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -220,3 +220,66 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('subnet', help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
         c.argument('vnet_name', help='Name of a virtual network.')
         c.argument('ip_address', help='IPv4 address or CIDR range.')
+
+    with self.argument_context('acr scope-map') as c:
```

Comment:
You don't need to split them per command here, instead a command group.

Reply:
What do you mean? I didn't understand it

Reply:
I see. Grouping them like this, everything will be auto-resolved depending on the params each function need. This is nice XD Changed.